scripts/linters/general_purpose_linter.py

```diff
@@ -536,10 +536,11 @@ def check_bad_pattern_in_file(filepath, file_content, pattern):
             or any(filepath.endswith(excluded_file)
                    for excluded_file in pattern['excluded_files'])):
```

Comment:
Are we expecting to have a new bad pattern in BAD_PATTERNS?

Reply:
@DubeySandeep Sorry, but I didn't get your comment.